examples/app/Embeddings.ipynb

```diff
@@ -20,36 +48,257 @@
+    }
+   },
+   "cell_type": "markdown",
+   "id": "7619577f",
+   "metadata": {},
+   "source": [
+    "You can change the type of the displayed weave panel by changing `Table` to `Plot` and it will intelligently change to display a plot of the data.\n",
+    "![image.png](attachment:image.png)"
```

Comment:
😆 documenting weave and you end up with a deep insight about Pokemon types

Reply:
I think we ignore the issue with types, I agree it's ugly. We should solve it by defaulting lists of text to be a table of vertical text panels rather than histograms.